cfpb/consumerfinance.gov
# Newsroom: Remove unneeded border and background-color

The newsroom has a double right border in the prefooter. This PR removes that redundant border, as well as the `related_content` class that is redundant to the background color set in `block__bg`.

## Removals

- `related_content ` class.
- `block__border-right` that is redundant to background set in `block__bg`.

## How to test this PR

1. Pull branch and `gulp clean && gulp build`.
2. Visit http://localhost:8000/about-us/newsroom/ and scroll to bottom and see that the prefooter has a background color and doesn't have a right-hand double border.


## Screenshots

Before, showing double border on right-hand side:
<img width="991" alt="Screen Shot 2021-10-12 at 10 51 00 AM" src="https://user-images.githubusercontent.com/704760/136981521-42a5d51d-3752-4d3b-9945-6d8b0395d367.png">
